specification/communication/data-plane/Chat/preview/2022-11-15-preview8/communicationserviceschat.json

```diff
@@ -0,0 +1,1977 @@
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "sendTypingNotificationRequest",
+            "description": "Details of the typing notification request.",
+            "schema": {
+              "$ref": "#/definitions/SendTypingNotificationRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful."
+          },
+          "401": {
+            "description": "Unauthorized.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "403": {
+            "description": "Forbidden.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "429": {
+            "description": "Too many requests.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "503": {
+            "description": "Service unavailable.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Post typing event to a thread": {
+            "$ref": "./examples/Messages_SendTypingNotification.json"
+          }
+        }
+      }
+    },
+    "/chat/threads/{chatThreadId}/{chatMessageId}/teamsinterop/image/{imageId}/$value": {
+      "get": {
+        "tags": [
+          "Image"
+        ],
+        "summary": "Get a teams interop image by id.",
+        "operationId": "TeamsInterop_GetImage",
+        "produces": [
+          "application/octet-stream",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "chatThreadId",
+            "description": "The thread id to which the message was sent.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "chatMessageId",
+            "description": "The message id.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "imageId",
+            "description": "The id of the image to download.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The action returns a stream to a image.",
+            "schema": {
+              "format": "binary",
+              "type": "file"
+            }
+          },
+          "401": {
```

Comment:
Instead of multiple error status codes we use one "default" case that returns an error. I know that the existing Chat API doesn't do that, but that's the current guideline.